# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1505819460 -7200
#      Tue Sep 19 13:11:00 2017 +0200
# Node ID 838fdcd4edf63dcf4c8569e0d31743abf3fbf8b5
# Parent  a03bb64031da9b6496b21ee338210eb74f46c552
Remove "getframe" from the list of unimplemented functions

* __unimplemented__.m: remove getframe

diff --git a/scripts/help/__unimplemented__.m b/scripts/help/__unimplemented__.m
--- a/scripts/help/__unimplemented__.m
+++ b/scripts/help/__unimplemented__.m
@@ -653,17 +653,16 @@ function list = missing_functions ()
   "fitsdisp",
   "fitsinfo",
   "fitsread",
   "fitswrite",
   "flow",
   "freqspace",
   "funm",
   "gammaincinv",
-  "getframe",
   "getpixelposition",
   "getReport",
   "gobjects",
   "graph",
   "graymon",
   "griddedInterpolant",
   "guide",
   "h5create",
